src/extensions/monitoring.js

```diff
@@ -814,81 +793,13 @@ const BacklogItemRow = ({
 }
 BacklogItemRow.displayName = 'BacklogItemRow'
 BacklogItemRow.propTypes = {
-  service: PropTypes.string.isRequired,
-  scanConfig: PropTypes.object,
   backlogItem: PropTypes.object.isRequired,
   selectedBacklogItems: PropTypes.arrayOf(PropTypes.object).isRequired,
   setSelectedBacklogItems: PropTypes.func.isRequired,
   refreshBacklogItems: PropTypes.func.isRequired,
 }
 
 
-const BacklogItemReference = ({
```

Comment:
I think this is a pretty neat feature, however I agree that extension specific special handling is quite cumbersome ...

Reply:
I am fine with removal for now, in case we will need something similar in future again, we should think about a more generic approach, e.g. add additional metadata to BLI to allow generic processing of something like external references (e.g. bdba URL)